torchvision/transforms/functional.py

```diff
@@ -105,9 +105,17 @@ def to_pil_image(pic, mode=None):
     Returns:
         PIL Image: Image converted to PIL Image.
     """
-    if not(_is_numpy_image(pic) or _is_tensor_image(pic)):
+    if not(torch.is_tensor(pic) or isinstance(pic, np.ndarray)):
```

Comment:
can we use instead `isisntance(pic, torch.Tensor)`? This is the recommended way now to checking if something is an instance of a Tensor or not

Reply:
Done!